test/sql/create_hypertable.sql

```diff
@@ -155,6 +155,52 @@ select create_hypertable('test_schema.test_1dim', 'time', if_not_exists => true)
 select create_hypertable('test_schema.test_1dim', 'time');
 \set ON_ERROR_STOP 1
 
+-- Test partitioning functions
+CREATE OR REPLACE FUNCTION invalid_partfunc(source integer)
+    RETURNS INTEGER LANGUAGE PLPGSQL IMMUTABLE AS
+$BODY$
+BEGIN
+    RETURN NULL;
```

Comment:
We should also test the case where an _valid_ time function returns NULL.

Reply:
Not sure what you mean. This function is named "invalid" because it returns NULL, which is not allowed. We don't support NULL values in time columns either.